src/wizards/tokenSwapDealWizard/stages/tokenDetailsStage/tokenDetailsStage.ts

```diff
@@ -127,8 +127,12 @@ export class TokenDetailsStage {
   }
 
   private addDefaultValuesToRegistrationData(wizardType: WizardType) {
-    if (wizardType === WizardType.createPartneredDeal) {
+    if (wizardType !== WizardType.createPartneredDeal) return;
```

Comment:
why are you changing this logic?  The original line is better in being simpler and not adding an additional point of return from the function.

Reply:
reverted